docs/development/deployment/chain-spec.md

```diff
@@ -2,24 +2,151 @@
 title: Chain Specification
 ---
 
-* What is a chain specification?
+A chain specification, or "chain spec", is a collection of configuration information that dictates
+which network a blockchain node will connect to, which entities it will initially communicate with,
+and what consensus-critical state it must have at genesis.
 
-* How do I launch a chain with a custom specification?
+## Structure of a Chain Spec
 
-* What does it mean to launch a substrate chain with a different chain spec?
+The [`ChainSpec` struct](https://substrate.dev/rustdocs/master/sc_service/struct.ChainSpec.html)
+separates the information contained in a chain spec into two parts. A node can use a `ChainSpec`
+instance to create a genesis block.
 
-* How do I generate one?
+### The Client Spec
 
-* What can be configured/changed?
+The first part of the chain spec, is the `ClientSpec`. The `ClientSpec` contains configuration
+information used by the Substrate client, the part of the node outside of the runtime. Much of this
+information is used to communicate with other parties in the network such as a set of bootnodes, a
+set of telemetry endpoints to which the node will send data, and human- and machine-readable names
+for the network to which the node will connect. Many of these items can be overridden by
+command-line flags, and the values can be changed after the blockchain has been launched.
 
-* Popular configurations
+> Caution: While all properties in this section can be changed after genesis, nodes will only add
+> peers who use the same `protocolId`.
 
-	* Initial Authorities
+### Extension
 
-	* Initial Balances
+Because the Substrate framework is extensible, it provides a way to customize the client spec with additional data to configure customized parts of the client. One example use case is telling the node about well-known blocks at specific heights, to prevent long range attacks when syncing a new node from genesis.
```

Comment:
```suggestion
Because the Substrate framework is extensible, it provides a way to customize the client spec with
additional data to configure customized parts of the client. One example use case is telling the node about
well-known blocks at specific heights, to prevent long range attacks when syncing a new node from genesis.
```